sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -936,7 +936,8 @@ class AdaptiveQueryExecSuite
           val error = intercept[SparkException] {
             joined.collect()
           }
-          assert(error.getMessage() contains "coalesce test error")
+          assert(error.getMessage().contains("coalesce test error") ||
+            error.getMessage().contains("Multiple failures in stage materialization"))
```

Comment:
For `Multiple failures in stage materialization`, we put each stage's failure into `Exception#suppressed`. Shall we find the `coalesce test error` in the suppressed errors?

Reply:
Sure, @cloud-fan .